frontend/lib/ui/Table/Table.stories.tsx

```diff
@@ -12,24 +12,51 @@ const data: [number, string, string][] = [
 ];
 
 export const BasicTable: Story = () => (
-  <Table id="basic_table">
-    <Table.Header>
-      <Table.Column>Number</Table.Column>
-      <Table.Column width={"20rem"}>Fixed width</Table.Column>
-      <Table.Column>Some value</Table.Column>
-    </Table.Header>
-    <Table.Body>
-      {data.map((row) => (
-        <Table.Row key={row[0]}>
-          <Table.Cell number fontSizeClass="fs-body">
-            {row[0]}
-          </Table.Cell>
-          <Table.Cell fontSizeClass="fs-md">{row[1]}</Table.Cell>
-          <Table.Cell fontSizeClass="fs-md">{row[2]}</Table.Cell>
-        </Table.Row>
-      ))}
-    </Table.Body>
-  </Table>
+  <>
+    <div className="mb-lg">
+      <Table id="basic_table">
+        <Table.Header>
+          <Table.Column>Number</Table.Column>
+          <Table.Column className="w-13">Fixed width</Table.Column>
+          <Table.Column>Some value</Table.Column>
+        </Table.Header>
+        <Table.Body>
+          {data.map((row) => (
+            <Table.Row key={row[0]}>
+              <Table.Cell className="table-number">{row[0]}</Table.Cell>
+              <Table.Cell>{row[1]}</Table.Cell>
+              <Table.Cell>{row[2]}</Table.Cell>
+            </Table.Row>
+          ))}
+        </Table.Body>
+      </Table>
+    </div>
+
+    <div className="mb-lg">
+      <Table id="styled_table">
+        <Table.Header className="bg-gray">
+          <Table.Column>Nummer</Table.Column>
+          <Table.Column>Stembureau</Table.Column>
+        </Table.Header>
+        <Table.Body className="fs-sm">
+          <Table.Row>
+            <Table.Cell className="table-number">33</Table.Cell>
```

Comment:
personal preference:
`<Table.NumberCell>` or `<Table.Cell isNumber>`

Reply:
I went too far indeed with the classes, the `<Table.NumberCell>` is really a different kind of cell with a lot of other styling.